spec/factories/tasks.rb

```diff
@@ -1,5 +1,11 @@
 FactoryBot.define do
   factory :task do
     name { "My Task" }
+
+    trait :with_three_nodes do
+      after(:create) do |task|
+         create(:queue, :with_three_nodes, queable: task)
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>[Correctable] Layout/IndentationWidth: Use 2 (not 3) spaces for indentation.